channels/tsmf/client/oss/tsmf_oss.c

```diff
@@ -34,7 +34,11 @@
 #include <libgen.h>
 #include <limits.h>
 #include <unistd.h>
-#include <sys/soundcard.h>
+#if defined(__OpenBSD__)
```

Comment:
This should be solved by including ```OSS_INCLUDE_DIR``` in CMake, then ```soundcard.h``` is in the include path.
See ```cmake/FindOSS.cmake```